app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    source_file, destination_path = parts[1], parts[2]
+
+    if os.path.exists(source_file):
+        if destination_path.endswith("/"):
```

Comment:
Do not use any os-specific directory delimiters in the code, your code should be platform-agnostic